TAJO-1677: Remove unnecessary messages for the Travis CI build

blrunner opened the pull request:
Enable -B (batch) mode on the build. This will make the logs shorter since it avoids the dependency download progress logging.

blrunner commented:
I've implemented this patch as follows.

- Apply -q parameter for maven :  Only show errors.
- Apply -ff parameter for maven : Stop at first failure in reactorized builds
- Apply surefire.useFile=false for maven: This causes surefire to print test failures to standard out.
- Clean up TestSystemMetrics log messages.
- Clean up TajoMaster log messages.
- Fix TestBSTIndexExec:testEqual error.
- Remove RestServer warning log messages.

hyunsik commented on tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java:
```diff
@@ -255,15 +255,15 @@ private void initWebServer() throws Exception {
       defaultFS.mkdirs(tajoRootPath, new FsPermission(TAJO_ROOT_DIR_PERMISSION));
       LOG.info("Tajo Root Directory '" + tajoRootPath + "' is created.");
+    } else {
+      LOG.info("FileSystem (" + this.defaultFS.getUri() + ") is initialized.");
```
This change seems to be different to the original intension. This log appears to show which HDFS namenode is used in this TajoMaster.

hyunsik commented on tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java:
```diff
@@ -280,19 +280,21 @@ private void checkAndInitializeSystemDirectories() throws IOException {
       defaultFS.mkdirs(wareHousePath, new FsPermission(WAREHOUSE_DIR_PERMISSION));
       LOG.info("Warehouse dir '" + wareHousePath + "' is created");
+    } else {
+      LOG.info("Tajo Warehouse dir: " + wareHousePath);
```
It should be recover due to the same reason as I commented above.

hyunsik commented on tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java:
```diff
@@ -280,19 +280,21 @@ private void checkAndInitializeSystemDirectories() throws IOException {
       defaultFS.mkdirs(stagingPath, new FsPermission(STAGING_ROOTDIR_PERMISSION));
       LOG.info("Staging dir '" + stagingPath + "' is created");
+    } else {
+      LOG.info("Staging dir: " + wareHousePath);
```
It should be recover due to the same reason as I commented above.

hyunsik commented on tajo-core/src/test/java/org/apache/tajo/engine/planner/physical/TestExternalSortExec.java:
```diff
@@ -98,8 +102,10 @@ public void setUp() throws Exception {
     appender.close();
 
-    System.out.println(appender.getStats().getNumRows() + " rows (" + (appender.getStats().getNumBytes() / 1048576) +
+    if (LOG.isDebugEnabled()) {
```
This if-condition branch seems to be not necessary.

hyunsik commented on tajo-core/src/test/java/org/apache/tajo/engine/planner/physical/TestProgressExternalSortExec.java:
```diff
@@ -102,7 +106,9 @@ public void setUp() throws Exception {
     appender.close();
 
-    System.out.println(appender.getStats().getNumRows() + " rows (" + appender.getStats().getNumBytes() + " Bytes)");
+    if (LOG.isDebugEnabled()) {
```
This if-condition branch seems to be not necessary.

hyunsik commented on tajo-core/src/test/java/org/apache/tajo/worker/TestNodeResourceManager.java:
```diff
@@ -253,7 +256,9 @@ public void run() {
                 }
               }
-              System.out.println(Thread.currentThread().getName() + " complete requests: " + complete);
+              if (LOG.isDebugEnabled()) {
```
This if-condition branch seems to be not necessary.

hyunsik commented on tajo-core/src/test/java/org/apache/tajo/worker/TestNodeResourceManager.java:
```diff
@@ -264,8 +269,10 @@ public void run() {
     }
 
-    System.out.println(parallelCount + " Thread, completed requests: " + totalComplete.get() + ", canceled requests:"
+    if (LOG.isDebugEnabled()) {
```
This if-condition branch seems to be not necessary.

hyunsik commented:
The first description mentioned that -B will be added to maven option. Was the plan changed?

hyunsik commented:
Please see the following volumes. quite.txt contains all messages from '-q' option. For me, `-q` seems to be not effective.
```
hyunsik@Hyunsiks-MacBook-Pro:~$ ls -l quite.txt normal.txt 
-rw-r--r--  1 hyunsik  staff  75536879 Jul 13 16:41 normal.txt
-rw-r--r--  1 hyunsik  staff  75490962 Jul 13 17:14 quite.txt
```

blrunner commented:
Hi @hyunsik 

Thank you for your detailed review.
I've updated the patch using your comments.
For your information, I added -q parameter instead of -B parameter. In some unexplained way, -B parameter didn't work as expect.

blrunner commented:
I think that the parameter on your desktop seemed like ineffective, because tajo dependencies already had been installed on your desktop. If you see build progress of this PR, you would see that there is no dependency download progress.

hyunsik commented on tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java:
```diff
@@ -202,7 +202,7 @@ public void serviceInit(Configuration _conf) throws Exception {
       tajoMasterService = new QueryCoordinatorService(context);
       addIfService(tajoMasterService);
-      
+
```
You need to recover meaningless changes.

hyunsik commented on tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java:
```diff
@@ -236,7 +236,7 @@ private void initSystemMetrics() {
   private void initResourceManager() throws Exception {
     Class<WorkerResourceManager>  resourceManagerClass = (Class<WorkerResourceManager>)
-        systemConf.getClass(ConfVars.RESOURCE_MANAGER_CLASS.varname, TajoWorkerResourceManager.class);
+      systemConf.getClass(ConfVars.RESOURCE_MANAGER_CLASS.varname, TajoWorkerResourceManager.class);
```
You need to recover meaningless changes.

hyunsik commented:
As I mentioned in some comments, most of changes are about coding format. You need to recover them. Coding format should be done in another issue.

blrunner commented:
Thanks @hyunsik 
I recovered codes related to coding format.

hyunsik commented:
+1 LGTM!
